athena/tests/modules/text/module_text_llm/mock/conftest.py

```
@@ -1,10 +1,22 @@
 # Import OpenAI mocks first to ensure they're in place before any other imports
-from tests.modules.text.module_text_llm.mock.utils.mock_openai import mock_openai, mock_openai_client
+from unittest.mock import patch
+from tests.modules.text.utils.mock_llm_config import mock_get_llm_config
+
+patch(
+    "llm_core.loaders.llm_config_loader.get_llm_config",
+    mock_get_llm_config,
+).start()
+
+from tests.modules.text.utils.mock_openai import mock_openai, mock_openai_client
```

Flake8 (7.2.0)

[error] 'tests.modules.text.utils.mock_openai.mock_openai' imported but unused

(F401)